modules/markup/markdown/markdown.go

```diff
@@ -296,3 +298,11 @@ func RenderRawString(ctx *markup.RenderContext, content string) (string, error)
 func IsMarkdownFile(name string) bool {
 	return markup.IsMarkupFile(name, MarkupName)
 }
+
+// Match hex colors
+var hexColorMatcher = regexp.MustCompile("(#[a-fA-f0-9]{6})")
```

Comment:
```suggestion
var hexColorMatcher = regexp.MustCompile("(#(?:[a-fA-f0-9]{8}|[a-fA-f0-9]{6}|[a-fA-f0-9]{4}|[a-fA-f0-9]{3}))(?:[^a-fA-F0-9]|$)")
```
`3`, `4`, and `8` char color codes are also possible, and no further color byte should follow to ignore false positives.